conduct/code_of_conduct.md

```diff
@@ -1,131 +1,159 @@
+We do not tolerate harassment or bullying of any community member in any form. 
+This applies equally to founders, developers, mentors and new community members, 
+in all spaces managed by Pangeo. This includes the mailing lists,
 our GitHub organizations, our chat rooms, in-person events, and any other
 forums created by the project team. In addition, violations of this code
 outside these spaces may affect a person's ability to participate within them.
 
-By embracing the following principles, guidelines and actions to follow or
-avoid, you will help us make Pangeo a welcoming and productive community. Feel
-free to contact the Code of Conduct Committee at
-[*TODO*](mailto:TODO) with any questions.
-
 
-1. **Be friendly and patient**.
+## Code of Conduct
 
-2. **Be welcoming**. We strive to be a community that welcomes and supports
+   
+* **1. Be friendly, patient, and welcoming**.    
+   We strive to be a community that welcomes and supports
    people of all backgrounds and identities. This includes, but is not limited
    to, members of any race, ethnicity, culture, national origin, color,
    immigration status, social and economic class, educational level, sex, sexual
    orientation, gender identity and expression, age, physical appearance, family
    status, technological or professional choices, academic
    discipline, religion, mental ability, and physical ability.
 
-3. **Be considerate**. Your work will be used by other people, and you in turn
+* **2. Be considerate and respectful**.    
+   Your work will be used by other people, and you in turn
    will depend on the work of others. Any decision you take will affect users
    and colleagues, and you should take those consequences into account when
-   making decisions. Remember that we're a world-wide community. You may be
+   making decisions.    
+   Remember that we're a world-wide community, and that you may be
    communicating with someone with a different primary language or cultural
-   background.
+   background. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners.
+   Do not insult or put down other community members. A community where people
+   feel uncomfortable or threatened is not a productive one.    
+   
+   
+* **3. Consider Impact vs. Intent**.    
+   Disagreements, both social and
+   technical, happen all the time and Pangeo is no exception. 
+   People perceive their own behaviors very differently than others perceive them. 
+   What motivates one person may cause stress for another. 
+   A few steps worth remembering include the following:
+
+   * Other people do not see things the same way you do, 
+   do not feel the same as you, or interpret the world the way you do. 
+   Consider different people’s perspectives on things like change, 
+   drive, structure, process, big picture, and small details.
+   * Ask for feedbacks and listen to them. 
```

Comment:
"feedback" (it's a non-countable noun) instead of "feedbacks" (which isn't a word).